src/libstore/optimise-store.cc

```diff
@@ -256,10 +257,16 @@ void LocalStore::optimisePath_(Activity * act, OptimiseStats & stats,
 
     stats.filesLinked++;
     stats.bytesFreed += st.st_size;
+#ifndef _WIN32
     stats.blocksFreed += st.st_blocks;
+#endif
```

Comment:
So apparently we're not using `blocksFreed` anywhere, so maybe we should just remove it.